erpnext/patches/v13_0/item_reposting_for_incorrect_sl_and_gl.py

```diff
@@ -6,10 +6,19 @@
 
 
 def execute():
-	for doctype in ('repost_item_valuation', 'stock_entry_detail', 'purchase_receipt_item',
-			'purchase_invoice_item', 'delivery_note_item', 'sales_invoice_item', 'packed_item'):
-		frappe.reload_doc('stock', 'doctype', doctype)
-	frappe.reload_doc('buying', 'doctype', 'purchase_receipt_item_supplied')
+	doctypes_to_reload = [
```

Comment:
Heh, should've used dictionary ideally as there are more doctypes than modules... thought about it but ultimately makes very little different as this patch likely won't change for a long time. 

Anyway I feel we need a better patch system to avoid this reloading stuff altogether.

Reply:
I don't think there is a good alternative to reloading. Remember: we still want the old data when patching..